modules/qualitycontrol/SendVideoController.js

```diff
@@ -74,8 +74,12 @@ export default class SendVideoController {
     _onMediaSessionStarted(mediaSession) {
         mediaSession.addListener(
             MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED,
-            session => {
-                if (session === this._conference.getActiveMediaSession()) {
+            (session, videoConstraints) => {
+                if (session === this._conference.getActiveMediaSession() && videoConstraints) {
```

Comment:
I think the old logic used to care about "is session the active session". Shouldn't it be the following?
```
if (session === this._conference.getActiveMediaSession()) {
    if (videoConstraints) {
     // source name logic
    } else {
     // legacy logic
    }
}
```

Reply:
I thought that the isActive media session chheck wasn't necessary since content-modify is sent only on p2p conn and it is always the active session if it exists. However, at the beginning of the call, p2p is not the active one until p2p ICE connection get established. I brought the check back.